Packs/CovalenceManagedSecurity/Integrations/CovalenceManagedSecurity/CovalenceManagedSecurity.py

```diff
@@ -221,6 +228,14 @@ def fetch_incidents(last_run, first_run_time_range):
                         incident['details'] += '\n\nMitigation Steps\n'
                         for step in a['steps']:
                             incident['details'] += f'''- {step['label']}\n'''
+                if org_id:
+                    active_response_profile = p.get_active_response_profile(org_id)
```

Comment:
Creating a new request for every incident might be a bit too much for the system. Is it up to you, but I would recommend checking this with a large number of incidents and making sure the system doesn't slow down too much and incidents are arriving at a reasonable pace.

Reply:
Thanks for your comment, we understand your concern.   
So far it's been ok, we'll keep it that way and if it becomes an issue, we might change how our API works to avoid multiple requests.